.github/workflows/ghpage.yaml

```diff
@@ -12,5 +12,19 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install mkdocs mkdocs-material
-      - run: mkdocs gh-deploy --force
+      - name: Building site
+        run: |
+          git checkout -b update_website
```

Comment:
That flow does not look right to me. From my point of view, we have the regular clone and then a second clone where we checkout gh-pages, branch from it to e.g. update_website. Build the docs in the first clone and copy them over to the second. Then open the PR for the second clone.